docs/v3.x/content-api/parameters.md

```diff
@@ -20,6 +20,8 @@ The available operators are separated in four different categories:
 
 ## Filters
 
+When using filters you can either pass simple filters in the root of the query parameters or pass them in a `_where` parameter.
```

Comment:
Is this being done to match up with GraphQL (I had mentioned doing something like this in my discussion parameters a few months back, but using `_filter` instead)

Reply:
https://github.com/strapi/strapi/discussions/4906#discussioncomment-5026

Subheader under `Querying` I do think `filter` is a bit more clean but I would understand the `where` if it's based on GraphQL as well.

Reply:
the end goal is to remove the root parameters and remove all the _ prefixing. Here we needed on parameter to support the nested queries and we were already using _where internally in the code so we kept it :)